docs/plat/hikey960.rst

```diff
@@ -146,7 +146,7 @@ Boot UEFI in recovery mode
        # The content of config file
        ./sec_usb_xloader.img 0x00020000
        ./sec_uce_boot.img 0x6A908000
-       ./l-loader.bin 0x1AC00000
+       ./recovery.bin 0x1AC00000
```

Comment:
In L157 below, please change to `Run the command to download recovery.bin into HiKey960`. Thanks!

Reply:
ok